programs/ziti-edge-tunnel/ziti-edge-tunnel.c

```diff
@@ -336,10 +339,13 @@ static bool process_tunnel_commands(const tunnel_command *tnl_cmd, command_cb cb
                 break;
             }
             log_level lvl = log_level_value_of(tunnel_set_log_level_cmd.loglevel);
-            if (lvl != log_level_Unknown) {
+            // int lvl = get_debug_level(tunnel_set_log_level_cmd.loglevel); // can be used only after c-sdk PR is merged
```

Comment:
Is there any reason to leave this here as a comment?

Also now that `lvl` is once again the integer mapping of a `log_level` enum, shouldn't the conditional test for `log_level_Unknown` instead of `-1`?

Reply:
I will remove this comment and make the necessary corrections in the ziti-edge-tunnel code after the c-sdk PR is reviewed and merged. If I make the changes now, it will give compilation error

Reply:
removed the comment